classes/components/listPanels/PKPDoiListPanel.php

```diff
@@ -172,6 +176,13 @@ public function getConfig()
             'manager.dois.update.failedCreation',
             'plugins.importexport.common.action.export',
             'manager.dois.actions.description',
+            'manager.dois.update.failedCreation',
+            'publication.version',
```

Comment:
Looks like this is only in OPS, so I'll move it over to pkp-lib. I'll probably wait until right before merge to avoid translation conflicts. @asmecher, could I give you a heads up then for you to pull in the latest translations?